pystac_client/client.py

```diff
@@ -394,7 +394,7 @@ def get_collection(self, collection_id: str) -> Union[Collection, CollectionClie
         Raises:
             NotFoundError if collection_id does not exist.
         """
-        collection: Union[Collection, CollectionClient]
+        collection: Collection | CollectionClient | None = None
```

Comment:
Got a full **pyupgrade** PR queued up here for merging after this one: https://github.com/stac-utils/pystac-client/pull/783

Reply:
oop yeah sorry I see those and can't resist